app/javascript/packs/hello_react.jsx

```diff
@@ -2,25 +2,25 @@
 // like app/views/layouts/application.html.erb. All it does is render <div>Hello React</div> at the bottom
 // of the page.
 
-import React from 'react'
-import ReactDOM from 'react-dom'
-import PropTypes from 'prop-types'
-
-const Hello = props => (
-  <div>Hello {props.name}!</div>
-)
-
-Hello.defaultProps = {
-  name: 'David'
-}
-
-Hello.propTypes = {
-  name: PropTypes.string
-}
-
-document.addEventListener('DOMContentLoaded', () => {
-  ReactDOM.render(
-    <Hello name="React" />,
-    document.body.appendChild(document.createElement('div')),
-  )
-})
+// import React from 'react'
+// import ReactDOM from 'react-dom'
+// import PropTypes from 'prop-types'
+//
+// const Hello = props => (
+//   <div>Hello {props.name}!</div>
+// )
+//
+// Hello.defaultProps = {
+//   name: 'David'
+// }
+//
+// Hello.propTypes = {
+//   name: PropTypes.string
+// }
+//
+// document.addEventListener('DOMContentLoaded', () => {
+//   ReactDOM.render(
+//     <Hello name="React" />,
+//     document.body.appendChild(document.createElement('div')),
+//   )
+// })
```

Comment:
When running `rake assets:precompile`, this file was throwing an error.  Everything else appears to be working without it, but I'm leaving this in in case Frontend needs this